ruby/example_code/glue/getting_started.rb

```diff
@@ -0,0 +1,216 @@
+    if job_run_status == "SUCCEEDED"
+      puts "Data from your job run is stored in your S3 bucket '#{@glue_bucket.name}'. Files include:"
+      begin
+
+        # Print the key name of each object in the bucket.
+        @glue_bucket.objects.each do |object_summary|
+          if object_summary.key.include?("run-")
+            print "#{object_summary.key}".yellow
+          end
+        end
+
+        # Print the first 256 bytes of a run file
+        desired_sample_objects = 1
+        @glue_bucket.objects.each do |object_summary|
+          if object_summary.key.include?("run-")
+            if desired_sample_objects > 0
+              sample_object = @glue_bucket.object(object_summary.key)
+              sample = sample_object.get(range: "bytes=0-255").body.read
+              puts "\nSample run file contents:"
+              print "#{sample}".yellow
+              desired_sample_objects -= 1
+            end
+          end
+        end
+      rescue Aws::S3::Errors::ServiceError => e
+        logger.error(
+          "Couldn't get job run data. Here's why: %s: %s",
+          e.response.error.code, e.response.error.message
+        )
+        raise
+      end
+    end
+    print "\nDone!\n".green
+
+    new_step(7, "Delete job definition and crawler.")
+    wrapper.delete_job(job_name)
+    puts "Job deleted: #{job_name}."
+    wrapper.delete_crawler(crawler_name)
+    puts "Crawler deleted: #{crawler_name}."
+    wrapper.delete_table(db_name, tables[0]["name"])
+    puts "Table deleted: #{tables[0]["name"]} in #{db_name}."
+    wrapper.delete_database(db_name)
+    puts "Database deleted: #{db_name}."
+    print "\nDone!\n".green
+  end
+end
+
+def main
+
+  banner("../../helpers/banner.txt")
+  puts "######################################################################################################".yellow
+  puts "#                                                                                                    #".yellow
+  puts "#                                          EXAMPLE CODE DEMO:                                        #".yellow
+  puts "#                                              AWS Glue                                              #".yellow
+  puts "#                                                                                                    #".yellow
+  puts "######################################################################################################".yellow
+  puts ""
+  puts "You have launched a demo of AWS Glue using the AWS for Ruby v3 SDK. Over the next 60 seconds, it will"
+  puts "do the following:"
+  puts "    1. Create a crawler."
+  puts "    2. Run a crawler to output a database."
+  puts "    3. Query the database."
+  puts "    4. Create a job definition that runs an ETL script."
+  puts "    5. Start a new job."
+  puts "    6. View results from a successful job run."
+  puts "    7. Delete job definition and crawler."
+  puts ""
+
+  confirm_begin
+  billing
```

Comment:
That's a clever little abstraction.

Reply:
Wonder if we could abstract it even further and share it with all languages 🤔

Reply:
Maybe we can abstract the text to a file and just read the file in each lang. Might be more trouble than it's worth, but could be an item for @Laren-AWS and the MVPeeps to discuss.